.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: Build and deploy
```

Comment:
The name is misleading because its only build not deployed

Reply:
right, fixed in 871674a